computer_vision/cnn_classification/cnn_classification.py

```diff
@@ -0,0 +1,91 @@
+"""
+Convolutional Neural Network
+
+Objective : To train a CNN model detect if TB is present in Lung X-ray or not.
+
+Resources CNN Theory : https://en.wikipedia.org/wiki/Convolutional_neural_network
+Resources Tensorflow : https://www.tensorflow.org/tutorials/images/cnn
+
+Download dataset from :
+https://lhncbc.nlm.nih.gov/LHC-publications/pubs/TuberculosisChestXrayImageDataSets.html
+
+"""
+
+# Part 1 - Building the CNN
+
+# Importing the Keras libraries and packages
+import tensorflow as tf
+from tensorflow.keras import layers, models
+
+import numpy as np
+
+
+# Initialising the CNN
```

Comment:
Could you wrap this entire part in `if __name__ == '__main__':` block as the code will only run if this script is called directly

Reply:
Okay. Done